flexx/app/_funcs.py

```diff
@@ -189,7 +189,12 @@ def init_notebook():
     
     # We set session_id and app_name in a way that it does not end up
     # in exported notebook.
-    url = 'ws://%s:%i/flexx/ws/%s' % (host, port, session.app_name)
+    if config.ssl_certfile != '' or config.ssl_keyfile != '':
+        # Connect over SSL
+        url = 'wss://%s:%i/flexx/ws/%s' % (host, port, session.app_name)
+    else:
+        url = 'ws://%s:%i/flexx/ws/%s' % (host, port, session.app_name)
```

Comment:
> Better use the same method as elsewhere, using server.proto
I guess the proto property returns 'http' / 'https', but we need 'ws' and 'wss'. Am I wrong?